src/arch/mod.rs

```diff
@@ -1,3 +1,6 @@
+use sysinfo::System;
```

Comment:
I don't think `arch` is the correct place for this code. This belongs probably in `vm.rs` .

Reply:
I changed this for now, but the reason why it intuitively made sense for me not to put `detect_freq_from_sysinfo` in `vm.rs` was the first place is because of this:

https://github.com/hermit-os/uhyve/blob/e9fc8ded286129d26097bcc5589c1f7d08e7f72c/src/vm.rs#L44

This made me unsure as to whether `vm.rs` is the right spot. If it isn't, it can be reverted.

Reply:
I changed this for now, but the reason why it intuitively made sense for me not to put `detect_freq_from_sysinfo` in `vm.rs` was the first place is because of this:

https://github.com/hermit-os/uhyve/blob/e9fc8ded286129d26097bcc5589c1f7d08e7f72c/src/vm.rs#L44

This made me unsure as to whether `vm.rs` is the right spot. If it isn't, it can be reverted.